### PR TITLE
Setup layout framework and example of implementation

### LINKED ISSUE #19: Layout/Template framework
Most pages share a common UI framework (header, footer et c) with other pages. This can be solved two ways.

1. The simplest way is using basic composition, where every page component contains a new instance of shared `<Header>` and `<Footer>` components.
2. Another way is "reverse composition" (for lack of a better name) where the page itself is contained within a higher-order component which also contains a `<Header>` and a `<Footer>`

Option 2 is the preferred solution.

The reason for going with option 2 is that it will keep the header and footer "alive" across pages. This allows for transitions, consistent scroll states et c, whereas option 1 would reset the header and footer on every navigation.

Another benefit is that overall layout/template code lives in a single location, instead of every page component. Adding a new feature  above the header only needs to happen in one place, not in every page.

The challenge with solution 2 is that different pages might have different layout requirements. The most obvious example is that organizer pages and public pages have vastly different layouts.

This means that the page component needs to somehow specify which layout to use, and the `App` component needs to use the correct one.

One approach to this is discussed here: https://github.com/zetkin/app.zetkin.org/discussions/1

### PR DESCRIPTION
**Pull request by kristofferlarberg**
Fixes #19

### CODE REVIEW

File: src/layout/DefaultLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { FunctionComponent } from 'react';
+
+const DefualtLayout : FunctionComponent = ({ children }) => (
```
Comment (richardolsson) [inline src/layout/DefaultLayout.tsx:3]: "Default" is misspelled here (but not in the module file name).

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -1,21 +1,31 @@
 import { AppProps } from 'next/app';
+import DefualtLayout from '../layout/DefaultLayout';
```
Comment (richardolsson) [inline src/pages/_app.tsx:2]: `DefaultLayout` is misspelled here (but then correctly spelled in the file name from which it's imported).

**Review by richardolsson — CHANGES_REQUESTED** [review]
Nice work! The structure is great. I especially like your use of composition to reuse the layout features of `DefaultLayout` (i.e. the "Zetkin" header) for the `OrgLayout`. I suspect that at a later time, we might want to actually flip these around. But lets cross that bridge when we get there.

I found some spelling mistakes in the code. Please fix those, then we're ready to merge!

**Comment by kristofferlarberg** [comment]
Thanks! Sorry about the misspellings, I found two more in _app. That's what you get for copy/paste i guess:)

**Review by richardolsson — APPROVED** [review]
LGTM!

**Pull request merged**